samples/client/petstore/java/jersey1/src/test/java/io/swagger/client/ApiClientTest.java

```diff
@@ -203,25 +246,12 @@ public void testParameterToPairsWhenValueIsEmptyStrings() throws Exception {
     }
 
     @Test
-    public void testParameterToPairsWhenValueIsNotCollection() throws Exception {
-        String name = "param-a";
-        Integer value = 1;
-
-        List<Pair> pairs = apiClient.parameterToPairs("csv", name, value);
-
-        assertEquals(1, pairs.size());
-        assertEquals(value, Integer.valueOf(pairs.get(0).getValue()));
-    }
-
-    @Test
     public void testParameterToPairsWhenValueIsCollection() throws Exception {
         Map<String, String> collectionFormatMap = new HashMap<String, String>();
         collectionFormatMap.put("csv", ",");
         collectionFormatMap.put("tsv", "\t");
         collectionFormatMap.put("ssv", " ");
-        collectionFormatMap.put("pipes", "\\|");
-        collectionFormatMap.put("", ","); // no format, must default to csv
-        collectionFormatMap.put("unknown", ","); // all other formats, must default to csv
```

Comment:
@michaelkourlas please put those test back when you've time.

I agree the behaviour is arguable but prefer to still keep the test cases before we change the behaviour.

> Don't we validate the spec before generating APIs from it first?

The current approach is to throw errors/warnings for invalid spec while trying to generate a functional clients. The goal is to provide a functional client that at least compiles right out of the box and the users can fix the spec or customize the setting (e.g. collection format) later.